### ISSUE TITLE
Update result record 

### ISSUE THREAD

**Issue by zednis** [tetherless-world/dtdi-mineral-observation-map#8]
The template in browser.html to generate HTML from search results needs to be updated.

```
    <script id="mineral-observation-template" type="text/x-handlebars-template">
        <tr>
            <td>
                <div>
                    <p>mineral observation record</p>
                    {{!-- Locality (e.g. "Camafuca-Camazambo Mine, Calonda, Cambulo City Council, Lunda Norte Province, Angola")--}}
                    {{!-- name --}}
                    {{!-- coordinates --}}
                    {{!-- link to MINDAT profile (badge?) --}}
                </div>
            </td>
        </tr>
    </script>
```

**Comment by zednis** [comment]
let's commit these changes into the currently open add-sample-json-search-document branch.

**Comment by zednis** [comment]
The current template uses HandlebarsJS: http://handlebarsjs.com/

The DCO person faceted browser HTML configuration uses this template system and can be used as a reference.

**Comment by zednis** [comment]
fixed by #6
